docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -26,102 +26,186 @@ Sometimes we just need to reboot the jetson device when it gets stuck in initial
+Then we have to move the pre-installed tensorrt package in Jetpack to the virtual env.
 
 First we use `find` to get where the tensorrt is
-```
+```shell
 sudo find / -name tensorrt
 ```
 Then copy the tensorrt to our destination like:
+```shell
+cp -r /usr/lib/python3.6/dist-packages/tensorrt* /your/path/to/archiconda3/envs/mmdeploy/lib/python3.6/site-packages/
 ```
-cp -r /usr/lib/python3.6/dist-packages/tensorrt* /home/archiconda3/env/mmdeploy/lib/python3.6/site-packages/
-```
-Meanwhle, tensorrt libs like `libnvinfer.so` can be found in `LD_LIBRARY_PATH`, which is done by Jetpack as well.
+Meanwhile, tensorrt libs like `libnvinfer.so` can be found in `LD_LIBRARY_PATH`, which is done by Jetpack as well.
 
-### Install torch
+### Install PyTorch
 
-Install the PyTorch for Jetsons **specifically**. Click [here](https://forums.developer.nvidia.com/t/pytorch-for-jetson-version-1-10-now-available/72048) to get the wheel. Before we use `pip install`, we have to install `libopenblas-base`, `libopenmpi-dev` first:
-```
-sudo apt-get install libopenblas-base libopenmpi-dev
+Before we use `pip install`, we have to install `libopenblas-base`, `libopenmpi-dev` first:
+```shell
+sudo apt-get install -y libopenblas-base libopenmpi-dev
 ```
 Or, it will throw the following error when we import torch in python:
 ```
 libmpi_cxx.so.20: cannot open shared object file: No such file or directory
 ```
+After that, download the PyTorch wheel for Jetson **specifically**. Click [here](https://forums.developer.nvidia.com/t/pytorch-for-jetson-version-1-10-now-available/72048) to get the wheel. MMDeploy is using `pytorch==1.8.0`.
+
+After the download finished, using cmd to install it.
+```shell
+pip3 install /your/path/dwonload/torch-1.8.0-cp36-cp36m-linux_aarch64.whl
+```
 
 ### Install torchvision
 We can't directly use `pip install torchvision` to install torchvision for Jetson Nano. But we can clone the repository from Github and build it locally. First we have to install some dependencies:
+```shell
+sudo apt-get install -y libjpeg-dev libpython3-dev libavcodec-dev libavformat-dev libswscale-dev
 ```
-sudo apt-get install libjpeg-dev libpython3-dev libavcodec-dev libavformat-dev libswscale-dev
-```
-Then just clone and compile the project:
-```
-git clone git@github.com:pytorch/vision.git
+Then just clone and compile the project, MMDeploy is using `torchvision=0.9.0`:
+```shell
+git clone https://github.com/pytorch/vision.git
 cd vision
-git checkout tags/v0.7.0 -b vision07
+git checkout tags/v0.9.0 -b v0.9.0
 pip install -e .
 ```
 
+**Note:** If the `pip install` throws error: `Illegal instruction (core dumped)`, then add `export OPENBLAS_CORETYPE=ARMV8` in `~/.bashrc` and then `source ~/.bashrc && conda activate mmdeploy` will solve the problem.
+
 ### Install mmcv
 
 Install openssl first:
+```shell
+sudo apt-get install -y libssl-dev
 ```
-sudo apt-get install libssl-dev
+Then install it from source, MMDeploy is using mmcv version is `1.4.0`:
```

Comment:
May use "Since MMDeploy is using mmcv `1.4.0`,  you can install it from source as below:"

Reply:
Done